client/my-sites/hosting/web-server-settings-card/index.js

```diff
@@ -34,6 +39,74 @@ const WebServerSettingsCard = ( {
 	const [ selectedPhpVersion, setSelectedPhpVersion ] = useState( '' );
 	const [ selectedStaticFile404, setSelectedStaticFile404 ] = useState( '' );
 
+	const getDatacenterOptions = () => {
+		return [
+			{
+				value: 'ams',
+				name: 'geo_affinity',
+				label: translate( 'Amsterdam' ),
+				thumbnail: {
+					imageUrl: '',
+				},
+			},
+			{
+				value: 'bur',
+				name: 'geo_affinity',
+				label: translate( 'California' ),
+				thumbnail: {
+					imageUrl: '',
+				},
+			},
+			{
+				value: 'dfw',
+				name: 'geo_affinity',
+				label: translate( 'Texas' ),
+				thumbnail: {
+					imageUrl: '',
+				},
+			},
+			{
+				value: 'dca',
+				name: 'geo_affinity',
+				label: translate( 'Washington, D.C.' ),
+				thumbnail: {
+					imageUrl: '',
+				},
+			},
+		];
+	};
+
+	const getGeoAffinityContent = () => {
+		if ( isGettingGeoAffinity ) {
+			return;
+		}
+
+		return (
+			<FormFieldset>
+				<FormLabel>{ translate( 'Primary Datacenter' ) }</FormLabel>
+				<FormSelect
+					disabled={ true }
+					className="web-server-settings-card__php-version-select"
+					value={ geoAffinity }
+				>
```

Comment:
If a prop is true, you can omit the value and pass only the prop name:

```suggestion
				<FormSelect
					disabled
					className="web-server-settings-card__php-version-select"
					value={ geoAffinity }
				>
```